qiskit/ignis/verification/entanglement/analysis.py

```diff
@@ -70,7 +94,11 @@ def title_maker(self):
 
         return title, title_ext
 
-    def sin_plotter(self, x, y, y_m=None):
+    def sin_plotter(self,
+                    x: np.array,
+                    y: np.array,
+                    y_m: np.array = None
+                    ):
         """
         Make sin plot of counts in both mqc and po exps.
```

Comment:
There's no `Args` sections for these. You've added types but no description of how they're used

Reply:
Done, thanks.